atomicapp/utils.py

```diff
@@ -214,3 +215,9 @@ def getRoot():
             return HOST_DIR
         else:
             return "/"
+
+    # generates a unique 12 character UUID
+    @staticmethod
+    def getUniqueUUID():
```

Comment:
Wouldn't it be better to generate a unique UUID with a combination of letters and numbers similar to Docker and rkt's way of generating ID's? Less chance for name colission if someone does multiple deploys.